airgun/entities/repository.py

```diff
@@ -61,9 +63,34 @@ def update(self, product_name, entity_name, values):
         if values.get('repo_content.http_proxy_policy') == 'Global Default':
             values['repo_content.http_proxy_policy'] = self.global_default_http_proxy
         view = self.navigate_to(self, 'Edit', product_name=product_name, entity_name=entity_name)
+        # Dismiss any existing flash messages before filling to avoid confusion
+        with suppress(NoSuchElementException, AttributeError):
```

Comment:
Is there actually any case where this is occuring? I'd prefer to not over error-proof this, it increases test run time and lowers code readibility.